troopers/alertify-bundle/3.0/config/packages/troopers_alertify.yaml

```diff
@@ -0,0 +1,13 @@
+#troopers_alertify:
```

Comment:
Please do not add configuration in comments like this. 
You should add some default config that is good for 80% of the users.

Reply:
actually, there is already a default config (https://github.com/Troopers/AlertifyBundle/blob/master/DependencyInjection/Configuration.php#L30), I added this to help user to overwrite the configuration as this is a frequent need.

Reply:
That is great!

But we should not bloat this configuration file with comments. Please add a link to your docs where the user can see how to configure the bundle. 

I would also like you to remove these config in comments before I can approve your recipe.

Reply:
I've made some changes to avoid comments and add the default config. The travis seems to fail because of twig dependancy in `framework.yaml`